lib/src/screen/dice/dice_screen.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class DiceScreen extends StatefulWidget {
+  DiceScreen({Key key, this.title})
+      : super(key: key);
+
+  final String title;
+
+
+  @override
+  _DiceScreenState createState() => _DiceScreenState();
+
+}
+
+class _DiceScreenState extends State<DiceScreen> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(widget.title),
+      ),
+      body: Center(
+        child: StreamBuilder(
```

Comment:
You are missing a stream here right?

Reply:
I removed the StreamBuilder without use.